script/test-acceptance

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -e
+
+export SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+export DEST=.
+
+bundle .integration-daemon-start
+
+apt-get install -y python-setuptools
+easy_install pip
+pip install virtualenv
```

Comment:
@shin- I would definitely put these steps in the `Dockerfile` and not there

- I don't want it to run each time I run `script/test-acceptance`
- I want to be able to run this script (outside the docker container, why not :stuck_out_tongue_closed_eyes:) on my Archlinux for example (but could be on a fedora, …) where, I don't have `apt-get`

These (`setuptools`, `pip` and `virtualenv`) are requirements to execute this test-acceptance, but they definitely shouldn't be in the script.